lib/utils.bash

```diff
@@ -27,22 +26,34 @@ sort_versions() {
 list_github_tags() {
 	git ls-remote --tags --refs "$GH_REPO" |
 		grep -o 'refs/tags/.*' | cut -d/ -f3- |
-		sed 's/^v//' # NOTE: You might want to adapt this sed to remove non-version strings from tags
+		grep -v sass | sed 's/^v//'
```

Comment:
There are a couple other releases, like `sass_api/<version>` and `sass-parser/<version>` that are also included in this repo, but we are not interested in, as we only want this to provide the `sass` executable. Filter them out here.